src/uTensor/core/context.hpp

```diff
@@ -52,12 +52,9 @@ class Context : public uTensor {
   //uint16_t getRef();
 
 public:
-  S_TENSOR add_static(std::function<void*(void)> func, TName _name);
-  S_TENSOR addCached(std::function<void*(void)> func, TName _name, uint8_t init_count = 0, bool _is_static = false);
```

Comment:
do you want to remove the static mode?

Reply:
Yep, as discussed, these APIs were never used in the codeine. And, life-cycle planning is better happened offline.